lib/sprout/src/main/kotlin/org/partiql/sprout/generator/target/kotlin/KotlinCommand.kt

```diff
@@ -56,10 +63,17 @@ class KotlinCommand : Callable<Int> {
         val input = BufferedReader(FileInputStream(file).reader()).readText()
         val parser = SproutParser.default()
         val universe = parser.parse(id, input)
+        if (restrictInterfaceImpl) {
+            // If interface implementations are restricted, add annotation to opt-in for generated sources.
+            // Currently only applies for the visitor interface.
```

Comment:
Currently, just apply the interface implementation restriction for the visitor interface